worker-workflow/src/test/java/com/github/cafdataprocessing/workflow/ArgumentsManagerTest.java

```diff
@@ -229,12 +229,9 @@ public void poisonDocumentHandlingTest() throws Exception {
         argumentsManager.addArgumentsToDocument(argumentDefinitions, document);
   
         final Type type = new TypeToken<Map<String, String>>() {}.getType();
-        final Map<String, String> arguments = gson.fromJson(
```

Comment:
Can we update the comment at the top of this test as well (line 196)? Thanks

Reply:
removed comment